### DISCUSSION TITLE
Segmentation fault(core dumped) when plot meshed model

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#204] [Q&A] [answered]
Dear Romain:
When I want to create a voxel-based microstructure then mesh it, and I want to plot the meshed model, there is the problem like the picture below:
![image](https://user-images.githubusercontent.com/75380898/116822864-8d9dec00-ab81-11eb-8a44-8cf2d24bff2a.png)

The code is like:

```
$ neper -T -n 1000 -domain "cube(1,1,1)" -morpho "gg,aspratio(2,1,0.5)" -ori random -tesrsize 128 -format tesr -o texture
$ neper -M texture.tesr -order 1 -elttype hex -rcl 1 -format msh,inp -statmesh nodenb -o texture
$ neper -V texture.msh -dataeltcol ori -print texture
```

And another question is that is there any simple relation(equation, formula, etc..) that between the command 'rcl' and 'cl' in Module-M? I use "grep -r rcl" but found so many things... If you can point out that's so nice!  

Thank you before and wish to hear your answer!
Best regards,
Damu

**Comment by rquey** [comment, marked as answer]
This is a bug, but if you use the tess format instead of the tesr format, it will work.

**Reply by damulaoshi** [reply]
Ok, thank you so much!
